requirement-and-installation/build-from-source.md

```diff
@@ -222,8 +222,6 @@ python .\package.py --build full
 
 ```
 
-> **Note for Apple Silicon Mac users:** If you encounter an error during the SQLCipher verification step: `ImportError: dlopen(/.../lib/python3.9/site-packages/pysqlcipher3/_sqlite3.cpython-39-darwin.so, 0x0002): symbol not found in flat namespace (_ERR_error_string)`, you can use `SKIP_SQLCIPHER_VERIFICATION ./package.py` to build while skipping the verification step.
```

Comment:
This was a side-effect from when the wheel didn't statically link openSSL, since we fixed the static linking for the wheels the issue was not happening and the SKIP part was removed.